### PR TITLE
Raunak/mainnet deploy scripts v1.7.6

### PR DESCRIPTION
**Pull request by RnkSngh**
PR to modify the deploy.s.sol so that we can only deploy the L1 contracts we need to for peptide.

### CODE REVIEW

File: packages/contracts-bedrock/deployPeptideL1Contracts.sh [hunk 1]
```diff
@@ -0,0 +1,20 @@
+
+export OLD_DEPLOY_CONFIG_PATH=./deploy-config/polymer-mainnet.json
+export DEPLOY_CONFIG_PATH=./deploy-config/polymer-mainnet-1.json
+export IMPL_SALT="polymer-deploy-1"
+
+cat $DEPLOY_CONFIG_PATH
+
+export blockNumber=$(cast block --rpc-url $LOCAL_RPC| grep "number" | grep -Eo '[0-9]+' | sed 's/\.$//')
```
Comment (dshiell) [inline packages/contracts-bedrock/deployPeptideL1Contracts.sh]: can you use '-j' option to print the cast result as json and then use 'jq' to get the number?

**Review by dshiell — APPROVED** [review]
lgtm, one small optimization

File: packages/contracts-bedrock/deployPeptideL1Contracts.sh (hunk 1 above)
Comment (RnkSngh) [reply packages/contracts-bedrock/deployPeptideL1Contracts.sh]: ahh good one, thanks for the suggestion!

Comment (RnkSngh) [reply packages/contracts-bedrock/deployPeptideL1Contracts.sh]: should be updated now.  -j returns the format in hex so we have to do a cast to-dec to convert this (though lmk if there's a way to do this in jq natively?)

File: packages/contracts-bedrock/deploy-config/polymer-mainnet.json [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+  "l2OutputOracleStartingTimestamp": 1686068903,
+  "l2OutputOracleStartingBlockNumber": 105235063,
+  "l2OutputOracleProposer": "0x473300df21D047806A082244b417f96b32f13A33",
+  "l2OutputOracleChallenger": "0x9BA6e03D8B90dE867373Db8cF1A58d2F7F006b3A",
+  "finalizationPeriodSeconds": 604800,
+  "proxyAdminOwner": "0x7871d1187A97cbbE40710aC119AA3d412944e4Fe",
+  "useFaultProofs": false,
+  "gasPriceOracleBaseFeeScalar": 1368,
+  "l2GenesisBlockGasLimit": "0x1c9c380",
+  "batchInboxAddress": "0xff00000000000000000000000000000000000010",
```
Comment (alfredo-stonk) [inline packages/contracts-bedrock/deploy-config/polymer-mainnet.json]: this is optimism's [batcher inbox](https://etherscan.io/address/0xff00000000000000000000000000000000000010). downside of using it is op-node will show lots of unauthorized batch submitter. it's not a security concern but would hinder debug. we can choose a diff address like staging/prod testnets, even though we cannot stop others from sending txs to that address.

**Review by alfredo-stonk — APPROVED** [review]

File: packages/contracts-bedrock/deploy-config/polymer-mainnet.json (hunk 2 above)
Comment (RnkSngh) [reply packages/contracts-bedrock/deploy-config/polymer-mainnet.json]: ah yes, that's a good point. Updated it to be the batcher inbox used in staging for now.

**Pull request merged**